modules/layers/src/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -79,7 +79,13 @@ export default class SolidPolygonLayer extends Layer {
 
     /* eslint-disable max-len */
     attributeManager.add({
-      indices: {size: 1, isIndexed: true, update: this.calculateIndices, noAlloc},
+      indices: {
+        size: 1,
+        isIndexed: true,
+        type: GL.UNSIGNED_INT,
```

Comment:
Why was this needed? Is specifying it here like this compatible with the 16 bit case? Short comment?

Reply:
If `type` is not specified, AttributeManager forces the external buffer to be Float32Array.